elements/tutorial/tutorialDialog.js

```diff
@@ -147,16 +147,11 @@ const winningCondition = (gameState, { level, isStoryModalOpen }, workspace, sen
 }
 
 function getNextLevelButton (send, level) {
-  if (level.nextTutorial) {
-    return buttonView({
-      label: 'Next Level',
-      onClick: () => send('location:set', `#tutorial/${level.nextTutorial}`)
-    })
-  }
-
   return buttonView({
-    label: 'Load Editor',
```

Comment:
Why did you remove the Load Editor Button, shouldn't we need that for the last tutorial ?